04/index.html

```diff
@@ -1,30 +1,34 @@
 <!DOCTYPE html>
 <html lang="pl">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>devmentor.pl - HTML & CSS: RWD - #04</title>
-</head>
-<body>
-    
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
-            </li>
-            <li>
-                <a href="#">oferta</a>
-                <ul>
-                    <li><a href="">strony internetowe</a></li>
-                    <li><a href="">pozycjonowanie stron www</a></li>
-                </ul>
-            </li>
-            <li>
-                <a href="#">kontakt</a>
-            </li>
-        </ul>
-    </nav>
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<meta http-equiv="X-UA-Compatible" content="ie=edge" />
+		<title>devmentor.pl - HTML & CSS: RWD - #04</title>
 
-</body>
-</html>
\ No newline at end of file
+		<link rel="stylesheet" href="./style.css" />
+	</head>
+	<body>
+		<nav class="nav">
+			<button class="nav__hamburger"></button>
+
+			<ul class="nav__list">
+				<li class="nav__list--item">
+					<a href="#home">Start</a>
+				</li>
+
+				<li class="nav__list--item menu__display">
+					<a href="#items">oferta<span class="arrow-right">▶</span></a>
+
+					<ul class="nav__under">
```

Comment:
Na wersji mobilne zagnieżdżenie powinno pokazać się poniżej. Inaczej przy małych ekranach kolejny poziom menu może być przesłonięty, a nawet w skrajnym przypadku niewidoczne:

![Screenshot 2024-04-19 at 06 13 44](https://github.com/devmentor-pl/practice-html-and-css-rwd/assets/55095257/89a4f961-e53b-4922-9597-2b3ddec1540d)